src/main/java/com/manywho/services/dummy/account/repositories/AccountRepository.java

```diff
@@ -36,26 +36,27 @@ public Account find(String id) {
     }
 
     public Account update(Account account) {
-        String sql = "UPDATE account SET name = :name, company = :company WHERE id = :id";
+        String sql = "UPDATE account SET name = :name, company = :company, stateid = :stateid WHERE id = :id and stateid = :stateid ";
```

Comment:
the same here, the id should be enough to identify the row that you are modifying